### DISCUSSION TITLE
Lesson 6    -  IndexError: list index out of range (Using Ganache-Cli)

### DISCUSSION THREAD

**Discussion by mdshaik07** [smartcontractkit/full-blockchain-solidity-course-py#687] [Q&A] [answered]
When i try to deploy script(deploy.py) after creating a new network locally ganache-local. I'm ending up with ::
**""** PS C:\Users\TV317DB\demos\brownie_fund_me> brownie run scripts/deploy.py --network ganache-local
INFO: Could not find files for the given pattern(s).
Brownie v1.17.2 - Python development framework for Ethereum

BrownieFundMeProject is the active project.

Running 'scripts\deploy.py::main'...
  File "C:\Users\TV317DB\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\_cli\run.py", line 50, in main
    return_value, frame = run(
  File "C:\Users\TV317DB\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\project\scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File ".\scripts\deploy.py", line 23, in main
    deploy_fund_me()
  File ".\scripts\deploy.py", line 6, in deploy_fund_me
    account = get_account()
  File ".\scripts\helpful.py", line 16, in get_account
    return accounts[0]
  File "C:\Users\TV317DB\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\network\account.py", line 115, in __getitem__
    return self._accounts[key]
IndexError: list index out of range. **""**

**Comment by cromewar** [comment]
Hello @mdshaik07 try deleting the network and add it again, look up for any typo, and remember the host name should start with http.

**Reply by mdshaik07** [reply]
Did it multiple times, brownie somehow is not throwing that error.

**Reply by cromewar** [reply]
The get account is returning accounts[0] which points to a ganache account and it's now finding it, could you please share you ganache ui configuration?

**Reply by mdshaik07** [reply]
i'm working with ganache cli, since ganache UI setup has security concerns with my organization. anyways and somehow, have resolved the issue... 
thanks @cromewar

**Reply by cromewar** [reply]
Oh that kind of things could get tricky sometimes, could you share please how did you solve it?

**Reply by mdshaik07** [reply]
I ran "ganache-cli" in one terminal.. and opened the other terminal to deploy script which get's connected to ganache-cli.
![image](https://user-images.githubusercontent.com/57830069/148324920-09ad1420-009b-41c9-aa42-d43eb5747c6c.png)

**Comment by mdshaik07** [comment, marked as answer]
I ran "ganache-cli" in one terminal.. and opened the other terminal to deploy script which get's connected to ganache-cli.

**Reply by cromewar** [reply]
Nice workaround 😄
